man/systemd.exec.xml

```diff
@@ -897,14 +897,14 @@
         in which case all paths listed will have limited access from within the namespace. If the empty string is
         assigned to this option, the specific list is reset, and all prior assignments have no effect.</para>
 
-        <para>Paths in <varname>ReadOnlyPaths=</varname> and <varname>InaccessiblePaths=</varname> may be prefixed with
-        <literal>-</literal>, in which case they will be ignored when they do not exist. Note that using this setting
-        will disconnect propagation of mounts from the service to the host (propagation in the opposite direction
-        continues to work). This means that this setting may not be used for services which shall be able to install
-        mount points in the main mount namespace. Note that the effect of these settings may be undone by privileged
-        processes. In order to set up an effective sandboxed environment for a unit it is thus recommended to combine
-        these settings with either <varname>CapabilityBoundingSet=~CAP_SYS_ADMIN</varname> or
-        <varname>SystemCallFilter=~@mount</varname>.</para></listitem>
+        <para>Paths in <varname>ReadWritePaths=</varname>, <varname>ReadOnlyPaths=</varname> and
+        <varname>InaccessiblePaths=</varname> may be prefixed with <literal>-</literal>, in which case they will be ignored
+        when they do not exist. Note that using this setting will disconnect propagation of mounts from the service to
+        the host (propagation in the opposite direction continues to work). This means that this setting may not be used
+        for services which shall be able to install mount points in the main mount namespace. Note that the effect of
+        these settings may be undone by privileged processes. In order to set up an effective sandboxed environment for
+        a unit it is thus recommended to combine these settings with either
+        <varname>CapabilityBoundingSet=~CAP_SYS_ADMIN</varname> or <varname>SystemCallFilter=~@mount</varname>.</para></listitem>
```

Comment:
Anyway, I can easy bypass `ProtectKernelTunables` and `ProtectControlGroups`
```
-bash-4.3# systemctl cat hola --no-pager
# /etc/systemd/system/hola.service
[Service]
ProtectKernelTunables=yes
CapabilityBoundingSet=~CAP_SYS_ADMIN
SystemCallFilter=~@mount
ProtectControlGroups=yes
ExecStart=/usr/bin/systemd-run sh -x -c 'echo HEY >/proc/sys/kernel/core_pattern'

-bash-4.3# echo OLD_VAL >/proc/sys/kernel/core_pattern
-bash-4.3# cat /proc/sys/kernel/core_pattern
OLD_VAL
-bash-4.3# systemctl start hola
-bash-4.3# cat /proc/sys/kernel/core_pattern
HEY
```
Do we really need these settings?